tests/containers/ecs_anywhere.pm

```diff
@@ -0,0 +1,62 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Create VM in EC2 using aws binary
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use registration;
+use testapi;
+use utils qw(zypper_call systemctl);
+use version_utils 'is_sle';
+use transactional qw(process_reboot trup_call);
+use registration qw(add_suseconnect_product get_addon_fullname);
+
+sub run {
+    my ($self, $args) = @_;
+    $self->select_serial_terminal;
+
+    systemctl 'enable --now docker.service';
```

Comment:
maybe we could call here `install_docker_when_needed` once https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14594 is merged